aten/src/ATen/KinetoPluginAPI.cpp

```diff
@@ -0,0 +1,32 @@
+#include "KinetoPluginAPI.h"
+
+#include <map>
+#include <vector>
+#include <mutex>
+
+namespace c10 {
```

Comment:
Is it reasonable to still put under c10 namespace?

Reply:
> Is it reasonable to still put under c10 namespace?

change to at namespace